zulip_bots/MANIFEST.in

```diff
@@ -1,47 +1,47 @@
-include zulip_bots/bots/github_detail/fixtures/test_pull.json
-include zulip_bots/bots/github_detail/fixtures/test_404.json
-include zulip_bots/bots/yoda/fixtures/test_invalid_input.json
-include zulip_bots/bots/yoda/fixtures/test_only_numbers.json
-include zulip_bots/bots/yoda/fixtures/test_2.json
-include zulip_bots/bots/yoda/fixtures/test_1.json
-include zulip_bots/bots/define/fixtures/test_multi_type_word.json
-include zulip_bots/bots/define/fixtures/test_single_type_word.json
-include zulip_bots/bots/define/fixtures/test_incorrect_word.json
-include zulip_bots/bots/wikipedia/fixtures/test_multi_word.json
 include zulip_bots/bots/wikipedia/fixtures/test_incorrect_query.json
-include zulip_bots/bots/wikipedia/fixtures/test_single_word.json
+include zulip_bots/bots/wikipedia/fixtures/test_multi_word.json
 include zulip_bots/bots/wikipedia/fixtures/test_number_query.json
-include zulip_bots/bots/xkcd/fixtures/test_specific_id.json
-include zulip_bots/bots/xkcd/fixtures/test_random.json
-include zulip_bots/bots/xkcd/fixtures/test_not_existing_id.json
+include zulip_bots/bots/wikipedia/fixtures/test_single_word.json
 include zulip_bots/bots/xkcd/fixtures/test_latest.json
+include zulip_bots/bots/xkcd/fixtures/test_not_existing_id.json
 include zulip_bots/bots/xkcd/fixtures/test_not_existing_id_2.json
+include zulip_bots/bots/xkcd/fixtures/test_random.json
+include zulip_bots/bots/xkcd/fixtures/test_specific_id.json
+include zulip_bots/bots/yoda/fixtures/test_1.json
+include zulip_bots/bots/yoda/fixtures/test_2.json
+include zulip_bots/bots/yoda/fixtures/test_invalid_input.json
+include zulip_bots/bots/yoda/fixtures/test_only_numbers.json
 
 include zulip_bots/bots/github_detail/logo.svg
 include zulip_bots/bots/googlesearch/logo.png
 
+include zulip_bots/bots/commute/doc.md
 include zulip_bots/bots/converter/doc.md
-include zulip_bots/bots/incrementor/doc.md
-include zulip_bots/bots/weather/doc.md
+include zulip_bots/bots/define/doc.md
+include zulip_bots/bots/encrypt/doc.md
+include zulip_bots/bots/foursquare/doc.md
 include zulip_bots/bots/github_detail/doc.md
-include zulip_bots/bots/yoda/doc.md
 include zulip_bots/bots/git_hub_comment/doc.md
-include zulip_bots/bots/virtual_fs/doc.md
+include zulip_bots/bots/googlesearch/doc.md
+include zulip_bots/bots/helloworld/doc.md
 include zulip_bots/bots/howdoi/doc.md
-include zulip_bots/bots/youtube/doc.md
+include zulip_bots/bots/incrementor/doc.md
 include zulip_bots/bots/john/doc.md
-include zulip_bots/bots/foursquare/doc.md
-include zulip_bots/bots/commute/doc.md
-include zulip_bots/bots/encrypt/doc.md
-include zulip_bots/bots/define/doc.md
-include zulip_bots/bots/helloworld/doc.md
 include zulip_bots/bots/tictactoe/doc.md
-include zulip_bots/bots/googlesearch/doc.md
+include zulip_bots/bots/virtual_fs/doc.md
+include zulip_bots/bots/weather/doc.md
 include zulip_bots/bots/xkcd/doc.md
+include zulip_bots/bots/yoda/doc.md
+include zulip_bots/bots/youtube/doc.md
```

Comment:
I'd like this file to eventually be added to `.gitignore`. Ideally, its not something we want future contributors to have to manually handle in any way before submitting a PR.